yb-voyager/cmd/fallForward.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import "github.com/spf13/cobra"
+
+var fallForwardCmd = &cobra.Command{
+	Use:   "fall-forward",
+	Short: "fall-forward",
+	Long:  `fall-forward`,
+}
+
+func init() {
+	rootCmd.AddCommand(fallForwardCmd)
+}
+
+func hideFlags(cmd *cobra.Command) {
```

Comment:
nit: since this function is specific for fall-forward command
`hideFlags` --> `hideFlagsInFallFowardCmds`